requirements.txt

```diff
@@ -6,3 +6,7 @@ Pillow==7.1.1
 python-decouple==3.3
 pytz==2019.3
 sqlparse==0.3.1
+
+
+drf-yasg
+
```

Comment:
```suggestion
drf-yasg
```

These empty extra lines are not needed here :)

Reply:
yes sir